README.md

```diff
@@ -1,4 +1,262 @@
 IPFS RFCs
 =====
 
-This repository will be the place where people propose, comment on, and refer to RFCs for changes to IPFS and its surrounding projects. Its initial manifestation will probably look a lot like the [Rust lang RFCs repository](https://github.com/rust-lang/rfcs)
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+  - [Opening](#ipfs-rfcs)
+  - [Table of Contents]
+  - [When you need to follow this process]
+  - [Before creating an RFC]
+  - [The RFC life-cycle]
+  - [Reviewing RFCs]
+  - [Implementing an RFC]
+  - [RFC Postponement]
+  - [Help this is all too informal!]
+  - [License]
+
+
+## When you need to follow this process
+[When you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make "substantial" changes to
+IPFS, libp2p or the RFC process itself. What constitutes a
+"substantial" change is evolving based on community norms and varies depending
```

Comment:
If "substancial" is not clear, we should give a pointer to who can be asked and is going to make a decision.